tests/functional/MapBrowseCest.php

```diff
@@ -9,11 +9,19 @@ public function _before(FunctionalTester $I)
     // tests
     public function tryToSeeCoordinatesInPageSource(FunctionalTester $I)
     {
+        // Get the coordinates in latitude and longitude format
+        $testFixture = array_map('str_getcsv', file('/var/www/data/dev/sample_attribute.csv'));
+        $expectPenguinCoordinate = $testFixture[5][3];
+        $expectFoxtailMilletCoordinate = $testFixture[14][3];
+
+        // Change coordinates in XY format by swapping the position
+        $expectPenguinCoordinateXY = implode(',', array_reverse(explode(',',$expectPenguinCoordinate)));
+        $expectFoxtailMilletCoordinateXY = implode(',', array_reverse(explode(',',$expectFoxtailMilletCoordinate)));
+
         $I->amOnPage("/site/mapbrowse");
         $I->seeResponseCodeIs(200);
-        $html =$I->grabPageSource();
-        $I->assertContains('"coordinates":[-45.295999 ,-60.345999]', $html, "Coordinate not found!");
-        $I->assertContains('"coordinates":[114.4698 ,38.0360]', $html, "Coordinate not found!");
-
+        $pageSource = $I->grabPageSource();
+        $I->assertContains($expectPenguinCoordinateXY, $pageSource, "Adelie penguin's coordinates not found!");
+        $I->assertContains($expectFoxtailMilletCoordinateXY, $pageSource, "Foxtail Millet's coordinates not found!");
```

Comment:
Hi @kencho51,
those three lines (23,24, 25) can be replaced with the more concise:

```
       $I->canSeeInSource($expectPenguinCoordinateXY);
       $I->canSeeInSource($expectFoxtailMilletCoordinateXY);
```

Also the HTTP response test at line 22 doesn't bring any value as if there is a non 200 response, the tests below will fail anyways and Codeception will generate snapshot that can show you what the error was. So it's better deleting that check.